### PR TITLE
Update Resiliency documentation for pod affinity and kubelet failure

### PR DESCRIPTION
**Pull request by alikdell**
Update Resiliency documentation for pod affinity and kubelet failure

# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| #https://github.com/dell/csm/issues/87
 #https://github.com/dell/csm/issues/165  |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shaynafinocchiaro — DISMISSED** [review]

**Comment by shaynafinocchiaro** [comment]
LGTM but can you merge into the `release-1.2` branch instead of `main`?

**Review by atye — DISMISSED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
it has to be release branch

File: content/docs/resiliency/_index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ CSM for Resiliency provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS | Supported Versions |
 |-|-|
-| Kubernetes    | 1.20, 1.21, 1.22 |
+| Kubernetes    | 1.20, 1.21, 1.22 1.23 |
```
Comment (tdawe) [inline content/docs/resiliency/_index.md:41]: Remove 1.20 if we are only supporting n-2 releases

**Review by atye — APPROVED** [review]

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by atye — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

File: content/docs/resiliency/_index.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ CSM for Resiliency provides the following capabilities:
 
 ## Supported CSI Drivers
 
-CSM for Authorization supports the following CSI drivers and versions.
```
Comment (shanmydell) [inline content/docs/resiliency/_index.md:57]: Good catch

**Pull request merged**